docs/.vuepress/config.js

```diff
@@ -115,8 +116,10 @@ module.exports = {
               sidebarDepth: 1,
               collapsable: false,
               children: [
-                ['/quickstart/publish','Publish with IPFS'],
+                ['/quickstart/publish','Publish with IPFS (UI)'],
+                ['/quickstart/publish_cli', 'Publish with IPFS (command line)' ]
```

Comment:
I ❤️ missing commas that break builds

```suggestion
                ['/quickstart/publish_cli', 'Publish with IPFS (command line)' ],
```